app/notification.go

```diff
@@ -639,6 +639,11 @@ func (a *App) userAllowsEmail(user *model.User, channelMemberNotificationProps m
 		}
 	}
 
+	// if user is a bot account, then we do not send email
+	if user.IsBot {
+		userAllowsEmails = false
```

Comment:
Should the return early with a  `false`?

Reply:
Sure, that makes sense! I will return early here so we do not have to go through the rest of the method if we are a bot user.